app/controllers/media_objects_controller.rb

```diff
@@ -242,6 +242,7 @@ def update_media_object
         if file_spec[:files].present?
           if master_file.update_derivatives(file_spec[:files], false)
             @media_object.ordered_master_files += [master_file]
+            master_file.update_stills_from_offset!
```

Comment:
Could you please add a test for this?  I think it could just expect that that method gets called and could probably live in this test: https://github.com/avalonmediasystem/avalon/blob/a8b38aaeac4364433c0d9988976e26fb09f0e457/spec/controllers/media_objects_controller_spec.rb#L413-L419